runtime/driver/src/main/java/org/apache/nemo/driver/NemoDriver.java

```diff
@@ -149,7 +153,18 @@ private void shutdown() {
     @Override
     public void onNext(final StartTime startTime) {
       setUpLogger();
-      runtimeMaster.requestContainer(resourceSpecificationString);
+      if (executorType.equals("lambda")) {
```

Comment:
I don't any difference here, because `lambda` is passed as an argument, and we won't expect other strings or any changes to the string here.
What do you say?

Reply:
for readability

Reply:
please address this comment